tools/utilities/coarsen-chkpt/CoarsenCheckpt.H

```diff
@@ -0,0 +1,37 @@
+#ifndef COARSENCHECKPT_H
+#define COARSENCHECKPT_H
+
+#include "amr-wind/incflo.H"
+
+namespace amr_wind {
+namespace tools {
+
+class CoarsenCheckpt : public incflo
+{
+public:
+    CoarsenCheckpt();
+
+    virtual ~CoarsenCheckpt() = default;
+
+    void read_chkpt_add_baselevel();
```

Comment:
and these are snake case ;)

Reply:
I bet it does look funny with the different cases! But this PR has a few tools in it. CoarsenCheckpt is separate from CheckpointToCSV. Each is different because of the code I started from.